packages/mdc-top-app-bar/mdc-top-app-bar.scss

```diff
@@ -130,5 +130,24 @@
   }
 }
 
+.mdc-top-app-bar--prominent {
+  .mdc-top-app-bar__row {
+    min-height: $mdc-top-app-bar-prominent-row-height;
```

Comment:
Why is this using min-height rather than height? Would something cause this to be even taller? And would we not be eventually interested in this collapsing to one line's height on scroll?